pgx-tests/src/tests/mod.rs

```diff
@@ -9,6 +9,7 @@ mod datetime_tests;
 mod default_arg_value_tests;
 mod derive_pgtype_lifetimes;
 mod enum_type_tests;
+mod extension_sql_tests;
```

Comment:
I think you may have forgotten to add a file, builds fail here trying to find the test file.